components/automate-ui/src/app/components/create-object-modal/create-object-modal.component.html

```diff
@@ -6,7 +6,7 @@ <h2 slot="title">Create {{ objectNoun | titlecase }}</h2>
           <chef-form-field>
             <label>
               <span class="label">Name <span aria-hidden="true">*</span></span>
-              <input chefInput formControlName="name" type="text" (keyup)="handleNameInput($event)" id="name-input" data-cy="create-name">
+              <input chefInput formControlName="name" type="text" (keyup)="handleNameInput($event)" id="name-input" data-cy="create-name" autocomplete="off">
```

Comment:
With the change to the library component all of these explicit additions still need to be removed, right...?

Reply:
As far as I've understood @scottopherson, `<chef-input>` and `<input chefInput>` are not the same thing. @scottopherson can you chime in, please?

Reply:
Sorry, was not more clear: What I meant was these items should be changed to `<chef-input>` so you can then remove the `autocomplete`.

Reply:
Hmm that also wasn't clear to me based on @scottopherson's answer -- **is** `<chef-input>` the clear winner we'll want to use everywhere?

Reply:
No, not a clear winner. Probably should find out what is not supported with `<chef-input>` but my understanding is that we should use it if we can use it.

Reply:
@msorens feel free to take over, but I'd rather get this in quick and do a cleanup refactoring later -- @susanev wanted to get this into the next promotion :)